srcpkgs/miniserve/template

```diff
@@ -1,15 +1,17 @@
 # Template file for 'miniserve'
 pkgname=miniserve
-version=0.14.0
+version=0.18.0
 revision=1
 build_style=cargo
 checkdepends="curl"
 short_desc="CLI tool to serve files and dirs over HTTP"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="MIT"
 homepage="https://github.com/svenstaro/miniserve"
+changelog="https://raw.githubusercontent.com/svenstaro/miniserve/master/CHANGELOG.md"
 distfiles="https://github.com/svenstaro/miniserve/archive/v${version}.tar.gz"
-checksum=68e21c35a4577251f656f3d1ccac2de23abd68432810b11556bcc8976bb19fc5
+checksum=24860f103879fd881a522a7ad8f25f2c9dcd5e01287b0485bcabf3e88c719015
+make_check=no  # tests pass locally but fail in CI
```

Comment:
```suggestion
make_check=ci-skip
```

Reply:
it would be also nice to describe what is happening if we know that

Reply:
I know that the tests fail because a file in `/tmp` isn't created properly (either it's named incorrectly or isn't created at all), but I don't know why that's happening.